labellab-client/src/reducers/auth.js

```diff
@@ -139,6 +142,32 @@ const auth = (state = initialState, action) => {
         passwordUpdatedMessage: action.payload,
         passwordUpdateError: true
       }
+      case OAUTH_LOGIN_REQUEST:
+        return {
+          ...state,
+          isAuthenticating: true
+        }
+      case OAUTH_LOGIN_SUCCESS:
+        return {
+          ...state,
+          isAuthenticated: true,
+          isAuthenticating: false,
+          statusText: 'You are logged in successfully!',
```

Comment:
Instead of hard coding the messages, just set them from backend. It will be good in future, if we want any changes in the message.

Reply:
okay sure!